src/jobs/run.yml

```diff
@@ -38,10 +38,23 @@ parameters:
             Optionally supply a custom package installation command, with any additional flags needed.
         type: string
         default: ''
+    resource_class:
```

Comment:
i don't think this resource class parameter needs to be here since it's already defined in the default executor. for example, tag isn't defined as a parameter in this specific job, but it is defined in the executor/default.yml

@KyleTryon - can you confirm?